client/my-sites/plugins/notices.jsx

```diff
@@ -979,69 +932,29 @@ class PluginNotices extends React.Component {
 				}
 
 			case 'RECEIVE_PLUGINS':
-				return i18n.translate( 'Error fetching plugins on %(site)s.', {
+				return translate( 'Error fetching plugins on %(site)s.', {
 					args: translateArg,
 				} );
 		}
 	};
 
-	getErrorButton( log ) {
-		if ( log.length > 1 ) {
-			return null;
-		}
-		log = log.length ? log[ 0 ] : log;
-		if ( log.error.error === 'unauthorized_full_access' ) {
-			return i18n.translate( 'Turn On.' );
-		}
-		return null;
-	}
-
-	getErrorHref( log ) {
-		if ( log.length > 1 ) {
-			return null;
-		}
-		log = log.length ? log[ 0 ] : log;
-		if ( log.error.error !== 'unauthorized_full_access' ) {
-			return null;
-		}
-		let remoteManagementUrl =
```

Comment:
I've been meaning to remove those `remoteManagementUrl` pieces anyway. Jetpack's support policy is current version -1, current version is 9.2, and Manage as a module was removed (and deemed always enabled) in Jetpack 7.3 (see https://github.com/Automattic/jetpack/pull/9089). So, this can be removed completely.

Now you'll notice that if we remove this, no errors have a button nor href anymore, so we can remove those methods and their usage altogether.